From 4d5afe96813822635fbebaba760cfa68f3c8a761 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Mon, 26 Nov 2012 17:31:49 -0800
Subject: [PATCH] mmc: xilinx_sdhcips: Correct minor style issues

Removing/adding braces where appropriate.
Remove unnecessary line continuation using '\'.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Acked-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/mmc/host/sdhci-of-xilinxps.c |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/mmc/host/sdhci-of-xilinxps.c b/drivers/mmc/host/sdhci-of-xilinxps.c
index 8bace08..d695c68 100644
--- a/drivers/mmc/host/sdhci-of-xilinxps.c
+++ b/drivers/mmc/host/sdhci-of-xilinxps.c
@@ -36,8 +36,8 @@ static struct sdhci_ops sdhci_zynq_ops = {
 };
 
 static struct sdhci_pltfm_data sdhci_zynq_pdata = {
-	.quirks = SDHCI_QUIRK_CAP_CLOCK_BASE_BROKEN | \
-			SDHCI_QUIRK_DATA_TIMEOUT_USES_SDCLK,
+	.quirks = SDHCI_QUIRK_CAP_CLOCK_BASE_BROKEN |
+		SDHCI_QUIRK_DATA_TIMEOUT_USES_SDCLK,
 	.ops = &sdhci_zynq_ops,
 };
 
@@ -58,8 +58,9 @@ static int __devinit sdhci_zynq_probe(struct platform_device *pdev)
 			host = platform_get_drvdata(pdev);
 			host->quirks |= SDHCI_QUIRK_BROKEN_CARD_DETECTION;
 		}
-	} else
+	} else {
 		printk("sdhci platform registration failed\n");
+	}
 
 	return ret;
 }
-- 
1.7.0.4

